Dockerfile

```diff
@@ -3,6 +3,18 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk --update add \
```

Comment:
I mean, sort of a nitpick ... but why not combine this with the next `apk --no-cache` step?  The `--update` here will cache the apk repos.